modules/tensor_mechanics/src/materials/HillElastoPlasticityStressUpdate.C

```diff
@@ -7,69 +7,83 @@
 //* Licensed under LGPL 2.1, please see LICENSE for details
 //* https://www.gnu.org/licenses/lgpl-2.1.html
 
-#include "ADHillElastoPlasticityStressUpdate.h"
+#include "HillElastoPlasticityStressUpdate.h"
 
 registerMooseObject("TensorMechanicsApp", ADHillElastoPlasticityStressUpdate);
+registerMooseObject("TensorMechanicsApp", HillElastoPlasticityStressUpdate);
 
+template <bool is_ad>
 InputParameters
-ADHillElastoPlasticityStressUpdate::validParams()
+HillElastoPlasticityStressUpdateTempl<is_ad>::validParams()
 {
-  InputParameters params = ADAnisotropicReturnPlasticityStressUpdateBase::validParams();
+  InputParameters params = AnisotropicReturnPlasticityStressUpdateBaseTempl<is_ad>::validParams();
   params.addClassDescription(
-      "This class uses the stress update material in a radial return isotropic power law creep "
+      "This class uses the stress update material in a radial return plasticity"
```

Comment:
This class uses the generalized radial return for anisotropic elasto-plasticity. Thanks for revising it since I had initially neglected to provide a good description.

Reply:
Updated.